xml/schema/Mailing/Mailing.xml

```diff
@@ -487,4 +487,14 @@
       <type>Select</type>
     </html>
   </field>
+  <field>
```

Comment:
I think if you move this right after created_date it will be in a more logical place in the table